apps/conf/credentials/cosign-cred.yaml

```diff
@@ -0,0 +1,10 @@
+name: cosign-cred-test
+secretName: cosign-keys-test
+namespaces:
+  - kyverno
+  - tekton-pipelines
+  - tek
+credentialEntity: cosign-test
+credentialIdentifier: signer-test
```

Comment:
add credential-type, value will be like "cosign", "randomkey"

Reply:
instead of credential entity field,Should I need to use credential-type